content/docs/reference/spec/migration/buildpack-api-0.4-0.5.md

```diff
@@ -0,0 +1,33 @@
++++
+title="Buildpack API 0.4 -> 0.5"
++++
+
+<!--more-->
+
+This guide is most relevant to buildpack authors. 
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.5) for buildpack API 0.5 for the full list of changes and further details.
+
+### Stack ID
+The buildpack may now indicate compatibility with any stack by specifying the special value `*`.
+
+### Character restrictions for process types
+For each process in `launch.toml`, `type` is now restricted to only contain numbers, letters, and the characters ., _, and -, so symlinks on both linux and windows filesystems can be created.
+
+### Cached layers
+Cached layers are now part of the list of Layers Types for clarity.
+ 
+### Override env vars
+Override is now the default behavior for env files without a suffix.
+It means that for the `/env/`, `/env.build/`, and `/env.launch/` directories, the default, suffix-less behavior will be `VAR.override` instead of `VAR.append`+`VAR.delim=:`.
+
+### Decouple Builpack Plan and BOM
+The Buildpack Plan and Bill-Of-Materials are now decoupled.
+The file in `/bin/build` that contains the Buildpack Plan entries for the buildpack is now read-only.
```

Comment:
By "the file" we mean "the `<plan[ER]>` positional argument", right? I think we might need to rephrase this because it reads weird.

Reply:
@jkutner, thanks for your feedback! I rephrase this sentence as part of https://github.com/buildpacks/docs/pull/279/commits/030d158ab700a4ad8b91f90b8465f4b085408655.
I'm going to resolve this conversation but please feel free to reopen it or open a new conversation if you think the new phrasing is still unclear.